ingestion/src/metadata/data_quality/validations/table/sqlalchemy/tableDiff.py

```diff
@@ -83,9 +84,9 @@ def build_sample_where_clause(
     reduced_concat = reduce(
         lambda c1, c2: c1.concat(c2), sql_alchemy_columns + [literal(salt)]
     )
-    sqa_dialect = create_engine(
+    sqa_dialect = make_url(
```

Comment:
I forgot if you have access to the runner where this is called, if so you can pass the dialect directly from the session `session.bind.dialect.name` I believe

Reply:
If you are referring to `self.runner_session` then I would rather avoid using that as it is a protected parameter and this validator does not make use of `sqlalchemy` except for compiling the where clause so I would avoid exposing it to interfaces it doesn't require.

Reply:
It's a different dialect for `table1` and `table2`. I prefer to have a centralized point for getting the dialect instead of using different methods for each table.